docs/ops/movement/ScatterElementsUpdate_12.md

```diff
@@ -0,0 +1,331 @@
+
+.. code-block:: cpp
+
+    output[indices[i]] = max(updates[i], output[indices[i]]) axis = 0
+
+- Update with mean value of the corresponding elements, reduction = "mean"
+
+.. code-block:: cpp
+
+    output[indices[i]] = mean(updates[i], output[indices[i]]) axis = 0
+
+
+For 2D tensor case, the update of the element corresponding to the ``[i][j]`` is performed as:
+
+.. code-block:: cpp
+
+    output[indices[i][j]][j] = reduction(updates[i][j], output[indices[i][j]][j]) if axis = 0
+    output[i][indices[i][j]] = reduction(updates[i][j], output[indices[i][j]][j]) if axis = 1
+
+Accordingly for 3D tensor case, the update of the element corresponding to the ``[i][j][k]`` is performed as:
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = reduction(updates[i][j][k], output[indices[i][j][k]][j][k]) if axis = 0
+    output[i][indices[i][j][k]][k] = reduction(updates[i][j][k], output[i][indices[i][j][k]][k]) if axis = 1
+    output[i][j][indices[i][j][k]] = reduction(updates[i][j][k], output[i][j][indices[i][j][k]]) if axis = 2
+
+**Attributes**:
+
+* *reduction*
+
+  * **Description**: The type of operation to perform on the inputs.
+  * **Range of values**: one of ``none``, ``sum``, ``prod``, ``min``, ``max``, ``mean``
+  * **Type**: `string`
+  * **Default value**: ``none``
+  * **Required**: *no*
+
+
+* *use_init_val*
+
+  * **Description**: Controls whether the elements in the data input tensor are used as init value for reduce operations.
+  * **Range of values**:
+    * true - data input elements are used
+    * false - data input elements are not used
+  * **Type**: boolean
+  * **Default value**: true
+  * **Required**: *no*
+  * **Note**: The attribute has no effect for *reduction* == "none"
+
+
+**Inputs**:
+
+*   **1**: ``data`` tensor of arbitrary rank ``r`` and of type *T*. **Required.**
+
+*   **2**: ``indices`` tensor with indices of type *T_IND*. The rank of the tensor is equal to the rank of ``data`` tensor. All index values are expected to be within bounds ``[-d, d - 1]`` along dimension ``d`` pointed by ``axis``. If multiple indices point to the same output location then the order of updating the values is undefined. Negative value of index means reverse indexing and will be normalized to value ``len(data.shape[axis] + index)``. If an index points to non-existing element then exception is raised. **Required.**
+
+*   **3**: ``updates`` tensor of shape equal to the shape of ``indices`` tensor and of type *T*. **Required.**
+
+*   **4**: ``axis`` tensor with scalar or 1D tensor with one element of type *T_AXIS* specifying axis for scatter. Negative ``axis`` means reverse indexing and will be normalized to value ``axis = data.rank + axis``. The value can be in range ``[-r, r - 1]`` where ``r`` is the rank of ``data``. **Required.**
+
+**Outputs**:
+
+*   **1**: Tensor with shape equal to ``data`` tensor of the type *T*.
+
+**Types**
+
+* *T*: any numeric type.
```

Comment:
Do we support boolean type? And what does reduction mean for this type?

Reply:
> In my opinion, boolean type shouldn't be a problem for `sum`, `prod`, `min`, `max` and it is supported case by frameworks, additional info can be added to the spec:
> 
> * For `boolean` type of `data` input, _reduction_ `sum`, `prod` behaves like logical `OR`, `AND` accordingly, but there is no implementation for `mean`.

Following my comment from the PR with ScatterElementsUpdate core implementation: https://github.com/openvinotoolkit/openvino/pull/18069#discussion_r1231767094, clarification for boolean type and mean mode has been added.